frontend/src/metabase/query_builder/components/filters/modals/BulkFilterList/BulkFilterList.styled.tsx

```diff
@@ -8,18 +8,14 @@ export const ListRoot = styled.div`
 `;
 
 export const ListRow = styled.div`
-  display: flex;
+  display: grid;
+  grid-template-columns: repeat(2, minmax(0, 1fr));
   padding: 0.375rem 0;
```

Comment:
Much as I love flexbox, I changed this from flex to css grid because I need to be able to reliably style certain filter content to be its own full line and `flex-wrap` wasn't cutting it, where grid span works perfectly.